scripts/battlefields/Riverne_Site_A01/ouryu_cometh.lua

```diff
@@ -0,0 +1,41 @@
+-----------------------------------
+-- Area: Riverne Site #A01
+-- Name: Ouryu Cometh
+-- !pos 732.550 -32.5 -506.544 30
+-----------------------------------
+local riverneID = zones[xi.zone.RIVERNE_SITE_A01]
+-----------------------------------
+
+local content = Battlefield:new({
+    zoneId        = xi.zone.RIVERNE_SITE_A01,
+    battlefieldId = xi.battlefield.id.OURYU_COMETH,
+    maxPlayers    = 18,
+    levelCap      = 75,
+    timeLimit     = utils.minutes(30),
+    index         = 0,
+    area          = 1,
+    entryNpc      = 'Unstable_Displacement',
+    exitNpc       = 'Spatial_Displacement',
+    requiredItems =
+    {
+        xi.item.CLOUD_EVOKER,
+        wearMessage = riverneID.text.TIME_LIMIT_FOR_THIS_BATTLE_IS + 2,
+        wornMessage = riverneID.text.TIME_LIMIT_FOR_THIS_BATTLE_IS + 1,
+    },
+    experimental = true,
+})
+
+content.groups =
+{
+    {
+        mobs      = { 'Ouryu' },
+        superlink = true,
+        death     = utils.bind(content.handleAllMonstersDefeated, content),
```

Comment:
This death handler will most likely need to be defined locally for when this BCNM has loot support since there is no armoury crate, but instead a direct drop to loot pool.  For now I would suggest just a comment and a function to set battlefield status to `WON`

Reply:
retested and everything still works as it should.

![image](https://github.com/LandSandBoat/server/assets/65316697/ee6055a1-441a-4ede-85a0-095508bef3d9)